README.md

```diff
@@ -53,8 +53,8 @@ Name of a service or service list to add or remove inbound access to. The
 service needs to be defined in firewalld.
 
 ```
-service: 'ftp'
-service: [ 'ftp', 'tftp' ]
+service: ftp
+service: [ ftp, tftp ]
```

Comment:
Is there a convention to use these spaces within brackets [` `ftp, tftp` `]?

Reply:
Well, now I can't find it.  But there is some recent version of ansible-lint or ansible-test that complains about `[ word, word ]` and suggests using `[word,word]`